docs/docs/development/local/repository.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+---
+# Repository overview
+
+## Codebase Structure
+
+```
+📁 .ddev
+📁 config
+📁 docs
+ |- 📁 docs
+ |- docusaurus.config.ts
+ |- sidebars.ts
+📁 src
+ |- 📁 frontend
+📁 tests
+ |- 📁 cypress
+    |- 📁 e2e
+📁 web
+```
+
+### [Local development](development/local) with DDEV
```

Comment:
FYI, locally I was seeing this link result in a 404 (it should just need a forward slash: `/development/local`)

Reply:
YOU ARE SO RIGHT! Would you be able to push that change? I'm in a different branch right now.